osi_hostvehicledata.proto

```diff
@@ -71,6 +75,15 @@ message HostVehicleData
     //
     optional VehicleLocalization vehicle_localization = 8;
 
+    // State of driver assistance systems.
+    //
+    // This can include:
+    //  - information presented to the driver, for example, parking sensors
+    //  - warnings raised by the vehicle, for example, forward collision warning
+    //  - corrective action taken by the vehicle, for example, auto emergency braking
+    //
+    repeated DriverAssistState driver_assist_state = 12;
```

Comment:
To stay consistent I would like to integrate this message in sth like
VehicleDrivingAssistance or VehicleAutomatedDriving.
Then there is space to grow beside just the state.